rasa/cli/utils.py

```diff
@@ -203,3 +205,38 @@ def payload_from_button_question(button_question: "Question") -> Text:
 def signal_handler(sig, frame) -> NoReturn:
     print("Goodbye 👋")
     sys.exit(0)
+
+
+def run_in_loop(f: Awaitable[T], loop: Optional[asyncio.AbstractEventLoop] = None) -> T:
```

Comment:
why do we need a Generic here? We know that it's gonna be a `Callable`, don't we?

Reply:
typevar is needed to properly set the return type. (annotation is wrong though, the callable part is missing :+1:, added that)